docs/migration_to_v1.md

```
@@ -1 +1 @@
-Documentation for this plugin moved to: https://capgo.app/docs/plugins/social-login/
\ No newline at end of file
+Documentation for this plugin moved to: https://capgo.app/docs/plugins/social-login/
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)